app/assets/stylesheets/views/mod-actions.scss

```diff
@@ -369,6 +369,10 @@
       flex-direction: column;
       padding: 0 var(--su-4);
       transition-duration: 1s;
+
+      &.hidden {
```

Comment:
Refactor (blocking): We have a CSS utility class [hidden](https://storybook.forem.com/?path=/story/utility-first-classes-display--hidden).

Reply:
We do but it's wasn't specific enough so was being overwritten.

Reply:
OK,  I've reworked it to use a `not` in order to keep specify in check.